_el/exercises.md

```diff
@@ -1,168 +1,95 @@
+
+*Συμπληρώστε τα κενά* είναι μια εκλεπτυσμένη μορφή Κ&Ε στην οποία δίνεται στον μαθητή κάποιο αρχικός κώδικας τον οποίο πρέπει να το ολοκληρώσει. (Στην πράξη, οι περισσότερες ασκήσεις Κ&Ε είναι στην ουσία "συμπληρώσετε τα κενά" επειδή ο δάσκαλος θα δώσει σχόλια για να υπενθυμίσει στους εκπαιδευόμενους τα βήματα που πρέπει να λάβουν υπόψη.) Όπως συζητείται στο [s:load](#CHAPTER), οι αρχάριοι συχνά αντιλαμβάνονται τις ασκήσεις "συμπληρώσετε τα κενά" λιγότερο εκφοβιστικές από ότι να γράψουν ολόκληρο τον κώδικα από την αρχή, και καθώς ο διδάσκων έχει δώσει ήδη το μεγαλύτερο μέρος της δομής της τελικής απάντησης, οι απαντήσεις είναι πολύ πιο προβλέψιμες και επομένως πιο εύκολο να ελεγχθούν.
+
+> **Συμπληρώστε τα κενά**
+>
+> Συμπληρώστε τα κενά ώστε ο παρακάτω κώδικας να εκτυπώσει τη συμβολοσειρά `'hat'`.
+>
 >     text = 'all that it is'
 >     slice = text[____:____]
 >     print(slice)
 
-As described in [s:load](#CHAPTER), [Parsons
-Problems](#g:parsons-problem) also avoid the "blank screen of terror"
-problem. The learner is given the lines of code needed to solve a
-problem, but has to put them in the right order. Research over the
-past few years has shown that Parsons Problems are effective because
-they allow learners to concentrate on control flow separately from
-vocabulary
-[[Pars2006](#CITE),[Eric2015](#CITE),[Morr2016](#CITE),[Eric2017](#CITE)]. The
-same research shows that giving the learner more lines than she needs,
-or asking her to rearrange some lines and add a few more, makes this
-kind of problem significantly harder [[Harm2016](#CITE)]. Tools for
-building and doing Parsons Problems online exist [[Ihan2011](#CITE)],
-but they can be emulated (albeit somewhat clumsily) by asking learners
-to rearrange lines of code in an editor.
-
-> **Parsons Problem**
-> 
-> Rearrange and indent these lines to sum the positive values in a list.
-> (You will need to add colons in appropriate places as well.)
-> 
+Όπως περιγράφεται στο [s:load](#CHAPTER), τα [Προβλήματα Parsons](#g:parsons-problems) βοηθούν επίσης στο να αποφευχθεί το πρόβλημα της "κενής οθόνης του τρόμου". Ο μαθητής λαμβάνει τις γραμμές κώδικα που απαιτούνται για την επίλυση ενός προβλήματος, αλλά πρέπει να τις τοποθετήσει στη σωστή σειρά. Έρευνες τα τελευταία χρόνια έχουν δείξει ότι τα Προβλήματα Parsons είναι αποτελεσματικά καθώς επιτρέπουν τους μαθητές να επικεντρωθούν στη ροή ελέγχου παρά στο λεξιλόγιο. [# CITE], [Eric2015] (# CITE), [Morr2016] (# CITE), [Eric2017] (# CITE)]. Οι ίδιες έρευνες έδειξαν ότι δίνοντας στον μαθητή περισσότερες γραμμές από όσες απαιτούνται, ή ζητώντας να αλλάξει κάποιες γραμμές και να προσθέσει μερικές ακόμα, κάνει αυτό το είδος προβλήματος σημαντικά δυσκολότερο. [[Harm2016] (# CITE)]. Υπάρχουν διαδικτυακά εργαλεία διαθέσιμα για την κατασκευή και εκτέλεση Προβλημάτων Parsons [[Ihan2011] (# CITE)], αλλά μπορούν εύκολα να εξοιμοιωθούν (έστω και κάπως άκομψα) ζητώντας από τους μαθητές να αναδιατάξουν τις γραμμές κώδικα σε ένα επεξεργαστή κειμένου.
```

Comment:
"κενής οθόνης ..."--> ελληνικά εισαγωγικά
επιτρέπουν τους μαθητές --> επιτρέπουν στους μαθητές
κάνει αυτό το είδος προβλήματος σημαντικά δυσκολότερο.--> αυτό το είδος προβλήματος γίνεται σημαντικά δυσκολότερο (χωρίς τελεία)
εξοιμοιωθούν--> εξομοιωθούν

Reply:
Thanks @pdelias ! Fixed in a new commit.